examples/dlint/color.rs

```diff
@@ -1,195 +1,318 @@
 // Copyright 2020-2021 the Deno authors. All rights reserved. MIT license.
 use crate::lexer::{lex, MediaType, TokenOrComment};
+use if_chain::if_chain;
+use pulldown_cmark::{Options, Parser, Tag};
 use std::convert::TryFrom;
 use swc_ecmascript::parser::token::{Token, Word};
 
-pub trait Colorize {
-  fn colorize(self) -> String;
+pub fn colorize_markdown(input: &str) -> String {
+  let mut options = Options::empty();
+  options.insert(Options::ENABLE_STRIKETHROUGH);
```

Comment:
Maybe we don't need more options for dlint, but what I did in dprint was:

```rust
options.insert(Options::ENABLE_TABLES);
options.insert(Options::ENABLE_FOOTNOTES);
options.insert(Options::ENABLE_STRIKETHROUGH);
options.insert(Options::ENABLE_TASKLISTS);
```

Reply:
thanks for the info. We'll enable other options once it's needed :)